DemocracyClub/yournextrepresentative
# Feature/add party nations

[Trello](https://trello.com/c/LfOf3CUa/3398-add-nation-breakdown-to-party-pages)

Peter requested that we add a breakdown of which nations parties field candidates in - this data is part of the EC's party dataset (see [here](https://search.electoralcommission.org.uk/English/Registrations/PP804) for example, "Fielding Candidates in England" etc.) and is available in their API.

Sym recommended that rather than storing these three fields separately we combine the data into a single field so that's the approach I've taken. 

To test locally:
- Run `manage.py migrate`
- Run `manage.py parties_import_from_ec`
- Run `manage.py runserver`
- Find non-joint parties on the GB register and the NI register, one each (In the next steps I've chosen specific parties but change it up if you like)
- Hit your local YNR API for the GB party, e.g.`http://localhost:8000/api/next/parties/PP53/` and check the `nations` field - this should be populated e.g. `["ENG","WAL","SCO"]`. 
    - The nations in the list should match whatever is on the EC register for that party under the three "Fielding candidates in x" fields.
    - EC register link e.g. https://search.electoralcommission.org.uk/English/Registrations/PP53
- Hit your local YNR API again for the NI party (e.g. `PP51`) and check the `nations` field - this should be `null`

Blocks https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1655

```[tasklist]
### PR Checklist
```
